src/libs/ReimbursementAccountUtils.js

```diff
@@ -3,15 +3,14 @@ import lodashGet from 'lodash/get';
 /**
  * Get the default state for input fields in the VBA flow
  *
- * @param {Object} reimbursementAccountDraft
  * @param {Object} reimbursementAccount
  * @param {String} fieldName
  * @param {*} defaultValue
  *
  * @returns {*}
  */
-function getDefaultStateForField(reimbursementAccountDraft, reimbursementAccount, fieldName, defaultValue = '') {
-    return lodashGet(reimbursementAccountDraft, fieldName, lodashGet(reimbursementAccount, ['achData', fieldName], defaultValue));
+function getDefaultStateForField(reimbursementAccount, fieldName, defaultValue = '') {
+    return lodashGet(reimbursementAccount, ['achData', fieldName], defaultValue);
```

Comment:
@tienifr @tylerkaraszewski I think we can remove this file and move it to the page component?